src/api/index.tsx

```diff
@@ -201,7 +201,7 @@ export const fetchBlocksTransactionsCChain = async (): Promise<loadBlocksTransac
 export async function loadValidatorsInfo() {
     return new Promise((resolve, reject) => {
         const urlValidators = `${getBaseUrl()}${baseEndpoint}/validatorsInfo`
-        var request = {
+        let request = {
```

Comment:
Thanks for considering my previous comment! :D

Reply:
It is better to consider using let :+1: